packages/presentation/lib/screens/login/data/login_data.dart

```diff
@@ -1,9 +1,30 @@
+import 'package:domain/exceptions/validation_exception.dart';
+
 class LoginData {
   final bool authFailure;
+  final ValidationExceptionStatus loginValidationStatus;
+  final ValidationExceptionStatus passwordValidationStatus;
 
   LoginData(
     this.authFailure,
+    this.loginValidationStatus,
+    this.passwordValidationStatus,
   );
 
-  const LoginData.init() : authFailure = false;
+  const LoginData.init()
+      : authFailure = false,
+        loginValidationStatus = ValidationExceptionStatus.ok,
+        passwordValidationStatus = ValidationExceptionStatus.ok;
+
+  LoginData copyWith({
+    bool? authFailure,
```

Comment:
why do we need this variable?

Reply:
To know if server side auth  ok or not

Reply:
and why do u need to separate them? from the ui side failed auth and failed local validators look in the same way